src/Forms/ConfirmedPasswordField.php

```diff
@@ -562,9 +571,33 @@ public function saveInto(DataObjectInterface $record)
             return;
         }
 
-        if (!($this->canBeEmpty && !$this->value)) {
-            parent::saveInto($record);
+        // Create a random password if password is blank
+        if ($this->canBeEmpty && !$this->value) {
+            $this->value = $this->createStrongRandomPassword();
         }
```

Comment:
Only do this for new `Member` records, not for all `ComfirmedPasswordField`s.

Reply:
Updated